yarl/__init__.py

```diff
@@ -798,6 +798,15 @@ def with_path(self, path, *, encoded=False):
         return URL(self._val._replace(path=path, query='', fragment=''),
                    encoded=True)
 
+    @staticmethod
+    def _query_var(v):
+        if isinstance(v, str):
+            return v
+        if type(v) is int:  # no subclasses like bool
```

Comment:
Identity comparison is confusing here.

Reply:
What is *identity* here?

Reply:
`id()`

Reply:
It's a type check. I don't think we need to change it.

Reply:
it's about `is` vs. `==`